### ISSUE TITLE
use from_geopandas for naturalearth regions?

### ISSUE THREAD

**Issue by mathause** [mathause/regionmask#116]

**Comment by aaronspring** [comment]
Do you think from_geopandas can be a cleaner and simpler implementation of NE defined regions? Is this what this issue is about?

**Comment by mathause** [comment]
Yes, currently there is a custom function to do this & it would be nice to use `from_geopandas` - but it's actually not super complicated.
